src/auxiliary/special_elixirs.jl

```diff
@@ -171,7 +173,8 @@ function insert_maxiters(expr)
       if is_plain_solve || is_trixi_solve
         # Do nothing if `maxiters` is already set as keyword argument...
         for arg in x.args
-          if arg isa Expr && arg.head === Symbol("kw") && arg.args[1] === Symbol("maxiters")
+          if arg isa Expr && arg.head === Symbol("kw") &&
+             arg.args[1] === Symbol("maxiters")
             return x
```

Comment:
I don't really like that a long `if` list uses the same indentation as the following code inside the `if` block

Reply:
I am more irked that it is *not8 using the same indentation but aligned at the opening of the if statement, i.e., it is a three-space indent 😱

Reply:
If one explicitly adds parens around it (before or after formatting), it looks ok again imho:
```julia
          if (arg isa Expr && arg.head === Symbol("kw") &&
              arg.args[1] === Symbol("maxiters"))
            return x
          end
```